apps/meteor/server/models/raw/Users.js

```diff
@@ -3121,4 +3121,20 @@ export class UsersRaw extends BaseRaw {
 			},
 		);
 	}
+
+	async countActiveUsersWithCustomRoles() {
```

Comment:
I'm also not comfortable with this one.. I'd suggest we remove it from this PR so we can go ahead with other metrics that are ok, and we visit this one in a separate PR, wdyt?